linux_os/guide/auditing/auditd_configure_rules/audit_rules_networkconfig_modification/oval/debian.xml

```diff
@@ -0,0 +1 @@
+ubuntu.xml
```

Comment:
I am not in favor of using symlinks. Instead of the symlink, please put a copy of the file. Or even better try to consolidate all existing OVAL checks into a single file.